Breeze.UI/src/app/wallet/advanced/feedback/feedback.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-feedback',
+  templateUrl: './feedback.component.html',
+  styleUrls: ['./feedback.component.css']
+})
+export class FeedbackComponent {
+    private _loading = false;
```

Comment:
Is it possible to use `enum` instead, or maybe limited type, like:
```
private state: ' errored' | 'success' | 'loading' | 'idle';
```
Or something similar. The problem with current approach is that you can potentially corrupt the state and have loading = true and success = true, which is incorrect as you can only be in one state at a time. Unless I am missing something.

Reply:
Can revisit this, but it's such a simple piece of work.  Will leave for now.